### PR TITLE
Update german translations for solidus_i18n

### PR DESCRIPTION
**Pull request by vasschneider**

### CODE REVIEW

File: config/locales/de.yml [hunk 1]
```diff
@@ -763,10 +771,12 @@ de:
     code: Code
     company: Firma
     complete: komplett
+    complete_order: Bestellung speichern
```
Comment (tvdeyen) [inline config/locales/de.yml]: I don't think this is the right term.

"Bestellung abschließen" would fit better here. Although technically we "save the order"  when you hit the button we should refer to what actually happens: we change the state to "complete"

File: config/locales/de.yml [hunk 2]
```diff
@@ -1148,44 +1161,44 @@ de:
     order_already_updated: Bestellung bereits geändert
     order_approved: Bestellung freigegeben
     order_canceled: Bestellung storniert
+    order_completed: Bestellung kostenpflichtig abgegeben
```
Comment (tvdeyen) [inline config/locales/de.yml]: The "kostenpflichtig" is only necessary for frontend users and we do not use this translation in the frontend.

This is only used in the backend flash notice and should better be called "Bestellung erfolgreich abgeschlossen" (so it fits the button on the confirm page)

File: config/locales/de.yml [hunk 3]
```diff
@@ -1239,7 +1252,7 @@ de:
     percent_per_item: Prozent pro Artikel
     permalink: Permalink
     phone: Telefon
-    place_order: Bestellung ausführen
+    place_order: Zahlungspflichtig bestellen
```
Comment (tvdeyen) [inline config/locales/de.yml]: 👍

File: config/locales/de.yml (hunk 1 above)
Comment (vasschneider) [reply config/locales/de.yml]: done

File: config/locales/de.yml (hunk 2 above)
Comment (vasschneider) [reply config/locales/de.yml]: done

**Comment by tvdeyen** [comment]
Build errors should be fixed by #99

**Comment by tvdeyen** [comment]
@vasschneider We merged #99 Could you please rebase your branch?

**Comment by vasschneider** [comment]
Thanks.

**Pull request merged**